charts/osm/values.yaml

```
@@ -99,6 +99,8 @@ OpenServiceMesh:
   useHTTPSIngress: false
   # -- Envoy log level is used to specify the level of logs collected from envoy
   envoyLogLevel: error
+  # -- Sets the max data plane connections allowed, 0 if disabled
```

Suggestion applies to all comments pertaining to this option.
```suggestion
  # -- Sets the max data plane connections allowed, set to 0 to not enforce limits
```